app/pages/admin/project-status.cjsx

```diff
@@ -146,9 +147,42 @@ VersionList = React.createClass
 ProjectStatus = React.createClass
   displayName: "ProjectStatus"
 
+  propTypes:
+    project: React.PropTypes.object.isRequired
+
+  getInitialState: ->
+    usedWorkflowLevels: []
+    workflows: []
+
   getDefaultProps: ->
     project: null
 
+  componentDidMount: ->
+    getWorkflowsInOrder @props.project, fields: 'display_name,active,configuration'
+      .then (workflows) =>
+        usedWorkflowLevels = @getUsedWorkflowLevels(workflows)
+        @setState usedWorkflowLevels: usedWorkflowLevels, workflows: workflows
+
+  onChangeWorkflowLevel: (workflow, event) ->
+    selected = event.target.value
+    if selected is 'none'
+      workflow.update({ 'configuration.level': undefined })
+    else
+      workflow.update({ 'configuration.level': selected })
+
```

Comment:
Is this section modifying `@state.workflows` without explicitly calling `@setState`?